packages/babel-helper-create-class-features-plugin/src/misc.js

```diff
@@ -84,12 +82,18 @@ export function injectInitialization(path, constructor, nodes, renamer) {
 
 export function extractComputedKeys(ref, path, computedPaths, file) {
   const declarations = [];
-
+  const state = {
+    classBinding: path.node.id && path.scope.getBinding(path.node.id.name),
+    file,
+  };
   for (const computedPath of computedPaths) {
-    computedPath.traverse(classFieldDefinitionEvaluationTDZVisitor, {
-      classBinding: path.node.id && path.scope.getBinding(path.node.id.name),
-      file,
-    });
+    if (computedPath.get("key").isReferencedIdentifier()) {
```

Comment:
nit: can we cache `computedPath.get("key")`? It is also used in `!computedPath.get("key").isConstantExpression()` later.

Reply:
@JLHwung Done. PTAL 🙂